src/Version.cpp

```diff
@@ -23,7 +23,7 @@ QString MumbleVersion::toString(unsigned int v) {
 }
 
 bool MumbleVersion::get(int *major, int *minor, int *patch, const QString &version) {
-	QRegExp rx(QLatin1String("(\\d+)\\.(\\d+)\\.(\\d+)"));
+	QRegExp rx(QLatin1String("(\\d+)\\.(\\d+)\\.(\\d+).(\\d+)"));
```

Comment:
> We maybe should make the build number part optional

The 4th part of the version is always present now.

> Furthermore if we extract the build number anyways we should make the function expose it just like the other values.

I agree.

Reply:
> The 4th part of the version is always present now.

But this function takes (in principle) an arbitrary String and thus I can't hurt to make it fool-proof in case someone passes a version with only 3 numbers :point_up:

Reply:
Also note that we use this function to parse the version passed via URL:
https://github.com/mumble-voip/mumble/blob/ffb2b70b32558c4db0619ea22a7c6865fba58dc2/src/mumble/MainWindow.cpp#L948-L949

This is definitely not guaranteed to follow the new scheme.

Reply:
Excellent point.

Reply:
The issue I was referring to was fixed in 148dc8d0e1107ff6b13184e9fc8abb35c73954fb